src/Core/Image/Parser/ImageTagSourceParser.php

```diff
@@ -56,13 +56,13 @@ public function parse($imageTag)
         $imageTag = preg_replace('/src="(\.\.\/|\.\/)+/', $replacement, $imageTag);
 
         if (null === $imageTag) {
-            return null;
+            return '';
```

Comment:
```suggestion
            return null;
```
Keep null and change the interface